apps/src/blockly/customBlocks/googleBlockly/behaviorBlocks.js

```diff
@@ -75,11 +76,12 @@ export const blocks = GoogleBlockly.common.createBlockDefinitionsFromJsonArray([
       'procedure_defnoreturn_set_comment_helper',
       'procedure_def_set_no_return_helper',
       'behaviors_block_frame',
+      'modal_procedures_no_destroy',
```

Comment:
This keeps the behavior definition block in the modal editor from being deleted?

Reply:
Update: I wondered why this didn't already exist, but it looks like we needed to add this when we added the procedure map, so assuming this all sounds right, no further questions!

Reply:
It was something I had to add for the [procedures](https://github.com/code-dot-org/code-dot-org/blob/f4896810ef156cbed830fab30f2aad0b88769022/apps/src/blockly/customBlocks/googleBlockly/proceduresBlocks.js#L166) once we were using the modal function editor. It prevents the procedure from being deleted from the procedure map for the editor workspace when we clear the editor workspace.